File: benchmarks/fib/fib.ts
function fib(n: number): number {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

const start = performance.now();
const result = fib(35);
const elapsed = performance.now() - start;
console.log(`fib(35) = ${result}`);
console.log(`Time: ${Math.round(elapsed)}ms`);

File: benchmarks/fib/fib.py
import time
def fib(n):
    if n <= 1: return n
    return fib(n-1) + fib(n-2)

start = time.time()
result = fib(35)
elapsed = (time.time() - start) * 1000
print(f"fib(35) = {result}")
print(f"Time: {elapsed:.0f}ms")
